counthits/reducer.py

```diff
@@ -1,29 +1,26 @@
 import sys
 
+oldKey = None
```

Comment:
According to convention:

> Function names should be lowercase, with words separated by underscores as necessary to improve readability.

More: https://www.python.org/dev/peps/pep-0008/#function-and-variable-names